client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -109,6 +109,9 @@ module.exports = React.createClass( {
 
 	render: function() {
 		var name = this.getProductName();
+		if ( this.props.cartItem.bill_period !== -1 ) {
```

Comment:
When we add a cart item to the cart on the client and POST it to the server, it has an undefined `bill_period` property before the response comes in. This means that we briefly display the "- annual period" suffix in the cart for e.g. premium theme purchases momentarily.

We can fix this by adding a presence check here:

```js
if( this.props.cartItem.bill_period && this.props.cartItem.bill_period !== 1 ) {
```

We might also be able to check `this.props.cart.hasLoadedFromServer`, which would be an improvement.